src/pages/policy/output/Prompt.jsx

```diff
@@ -1,20 +1,28 @@
 import Button from "../../../controls/Button";
 import Screenshottable from "../../../layout/Screenshottable";
 import { PythonCodeBlock } from "./PolicyReproducibility";
+import { useSearchParams } from "react-router-dom";
 
-export default function Analysis(props) {
+
+export default function Prompt(props) {
   const { impact, policyLabel, metadata, policy } = props;
+  const [searchParams] = useSearchParams();
+  const selectedVersion = searchParams.get("version") || metadata.version;
   const relevantParameters = Object.keys(policy.reform.data).map(parameter => metadata.parameters[parameter]);
-  const policyDetails = `I'm a researcher at PolicyEngine, and I'm writing up an economic analysis of a tax-benefit policy reform. Please write the analysis for me using the details below, in their order. You should:
+  const policyDetails = `I'm a researcher using PolicyEngine, a free, open source tool to compute the impact of public policy. I'm writing up an economic analysis of a tax-benefit policy reform. Please write the analysis for me using the details below, in their order. You should:
   
-  * First explain each provision of the reform in detail.
+  * First explain each provision of the reform in detail, noting that it represents policy reforms for 2023.
```

Comment:
can you replace 2023 with the selected year? i couldn't see how

Reply:
Yep done- also added the region.